Week1/connection-test.js

```diff
@@ -3,7 +3,9 @@ var connection = mysql.createConnection({
   host     : 'localhost',
   user     : 'hyfuser',
   password : 'hyfpassword',
-  database : 'userdb'
+  database : 'userdb',
+  port : 3307 // this line was not added before to the file. I added according to my selected port. It could be changed according to your selected port
```

Comment:
The default port is 3306 and by default used by [mysqljs](https://github.com/mysqljs/mysql/blob/3430c513d6b0ca279fb7c79b210c9301e9315658/Readme.md)

I would suggest adding the port option as the comment and writing something like
`// port : xxxx // Uncomment this line and replace xxxx with the selected port number if you are not using default 3306`